crtclient/docker/docker.go

```diff
@@ -75,7 +75,7 @@ func (d *Docker) getContPid(ctx *crtclient.ContainerEpContext) (string, error) {
 	contInfo, err := d.Client.InspectContainer(contNameOrId)
 	if err != nil {
 
-		log.Printf("unable to get container info for '%s' \n",
+		log.Errorf("unable to get container info for '%s' \n",
```

Comment:
yeah, that's a good idea. logrus provides a WithFields() logging method that can give structure to our log messages as a starter, https://github.com/Sirupsen/logrus#fields